main.go

```diff
@@ -230,60 +245,65 @@ func filterApps(allFiles []parser_struct.UpdatedRepo, appSpec godo.AppSpec) AllE
 	for val := range allFiles {
 		nameMap[allFiles[val].Name] = true
 	}
-	//For future dockerhub integration first we need to update input json file and add parameter to specify dockerhub image and registry name
-	//then we can update code shown below to update app spec with dockerhub image
 
-	//iterate through all services, worker and job to update DOCR image in AppSpec based on unique name declared in input
+	//iterate through all services, worker and job to update AppSpec.ImageSourceSpec based on component name declared in input
 	for key := range allFiles {
 		for _, service := range appSpec.Services {
 			if service.Name != allFiles[key].Name {
 				continue
-			} else {
-				repos := strings.Split(allFiles[key].Repository, `/`)
```

Comment:
Since we were leaving these fields in the struct I think we should still parse this legacy format for backwards compatibility.

Reply:
We do parse it https://github.com/ElanHasson/app_action/commit/0731630645edaf22d083650882335a941a922fd2 still.

Did you mean something else?